overlord/snapstate/handlers.go

```diff
@@ -1153,6 +1234,7 @@ func (m *SnapManager) doLinkSnap(t *state.Task, _ *tomb.Tomb) (err error) {
 	t.Set("old-devmode", oldDevMode)
 	t.Set("old-jailmode", oldJailMode)
 	t.Set("old-classic", oldClassic)
+	t.Set("old-lastactivedisabledservices", svcsToSave)
```

Comment:
Looking at other key names, we could use `old-last-active-disabled-services`

Reply:
Done